_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
-- [Annex 5: Guidance](#annex-5-guidance)
-  - [Use of Open Source](#use-of-open-source)
-- [References](#references)
-
-## About this document
-
-The creation of this document was requested by the Director General of Strategy, Architecture and Business Relationships (SABR) within the Innovation, Information and Technology Branch (IITB) within Employment and Social Development Canada (ESDC) to guide future technical software procurement and implementation decisions at the enterprise level.
-
-This strategy focuses on accounting for risk within the build versus buy decision, and emphasizes that solutions are not built nor bought; components are
-One must break down a solution into its components through solutions architecture in order for the build versus buy decisions to be made for each component, taking risk of mandate delivery in the decision for each component.
-
-### Intended Audience
-
-This document is intended as guidance for the Enterprise Architecture Review Board (EARB).
-
-## The Strategy
-
-### Buy vs Build: oversimplifying solutions architecture
-
-Looking for a systemic and overarching approach to buy vs build of solutions at the organizational level means bypassing the whole concept of solution architecture, where the required experts of the organization, both from business lines, finance, security, information technology, etc., come together to figure out the best possible way of accomplishing its mission through leveraging its various resources including people, tools, processes and rules.
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+  - [Governance, Compliance, and Reporting](#governance-compliance-and-reporting)
+  - [Finance](#finance)
+  - [Architecture](#architecture)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
+- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix D - Definitions](#appendix-d---definitions)
+- [Appendix C - Context](#appendix-c---context)
+- [Appendix E - References](#appendix-e---references)
+
+## Introduction
+
+### Purpose
+
+To provide IITB with a build versus buy assessment framework.
+
+The strategy includes:
+
+1. A **guiding policy**, which serves to set automatic decisions that defines the build versus buy assessment framework
+2. A **coherent set of actions** (an action plan), which serve to operationalize the build versus buy assessment framework.
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
```

Comment:
Recommend adding reference to EARB as the intended audience, as that was explicitly the target audience for this strategy from our mgmt

Reply:
I've added a line in the Target Audience that stakeholders would also include guiding committees and boards. Line number 76.